.github/workflows/deploy-preview.yml

```
@@ -0,0 +1,61 @@
+name: "Deploy Preview"
+
+on:
+  workflow_run:
+    workflows: [Website Deploy Preview]
+    types:
+      - completed
+
+env:
+  AWS_REGION: us-east-2
+  IAM_ROLE_ARN: arn:aws:iam::068007702576:role/cplive-plat-ue2-dev-atmos-docs-gha
+  IAM_ROLE_SESSION_NAME: cloudposse-atmos-ci-deploy-pr-${{ github.event.workflow_run.pull_requests.0.number }}
```

actionlint (1.7.4)

got unexpected character 'n' while lexing fraction part of float number, expecting '0'..'9'

(expression)